docs/hub/mlx-image.md

```diff
@@ -0,0 +1,89 @@
+# Using mlxim at Hugging Face
+
+[`mlxim`](https://github.com/riccardomusmeci/mlx-image) is an image models library built on Apple [MLX](https://github.com/ml-explore/mlx). It tries to replicate the great [timm](https://github.com/huggingface/pytorch-image-models) library from Ross Wightman, but for MLX models.
+
+
+## Exploring MLX on the Hub
+
+You can find `mlxim` models by filtering at the left of the [models page](https://huggingface.co/models?library=mlxim&sort=trending).
+There's also an open [mlx-vision](https://huggingface.co/mlx-vision) space for contributors converting and publishing weights for MLX format.
```

Comment:
I don't see a way for people to join the community. Perhaps you want to consider these options? (taken from the settings of `mlx-community`)

<img width="703" alt="Screenshot 2024-03-11 at 10 47 29" src="https://github.com/huggingface/hub-docs/assets/1177582/64480970-7e3d-4f2b-a5c5-0903c350860f">

Reply:
you were right.. now it's fixed.